### ISSUE TITLE
SQL "Oh Noes" Errors

### ISSUE THREAD

**Issue by EagleBlizz** [Khan/live-editor#632]
On certain errors on SQL projects, Oh Noes underlines the wrong line and therefore leads the programmer into a state of confusion. Some examples:

https://www.khanacademy.org/computer-programming/spin-off-of-khan-academy-longest-streak-leaderboard-with-bug-2/6657580983713792
`Line 5` should be underlined marking the `Day Streak` separation.

https://www.khanacademy.org/computer-programming/spin-off-of-khan-academy-longest-streak-leaderboard-with-bug-1/6523028248264704
`Line 8` should be underlined marking the `;` (instead of `,` ) error.

Best regards!

**Comment by kevinbarabash** [comment]
I agree the first bug could be better reported, but it is in the general vicinity.  The second bug reports something different for me.  Error buddy says: `There's a syntax error near (` on line 9.
